constraints.txt

```diff
@@ -0,0 +1,2 @@
+git+git://github.com/david4096/ga4gh-client.git@protobuf31#egg=ga4gh_client
```

Comment:
We could add a script that before a release adds the pinned versions and then runs register/upload.

The main thing here is whether we are ok with having the development version pointing at schemas/master as a dependency. This has the nice feature that changes in the schemas are breaking in the server. Although the schemas shouldn't move forward without a working implementation, this gives us better guarantees of that dependency in code.